rules/falco_rules.yaml

```diff
@@ -2363,7 +2363,9 @@
   condition: >
     (container.image.repository in (gcr.io/google_containers/hyperkube-amd64,
      gcr.io/google_containers/kube2sky, sysdig/agent, sysdig/falco,
-     sysdig/sysdig, falcosecurity/falco) or (k8s.ns.name = "kube-system"))
+     sysdig/sysdig, falcosecurity/falco, registry.ng.bluemix.net/armada-master/olm,
+     icr.io/ext/sysdig/agent:latest, registry.ng.bluemix.net/armada-master/metrics-server-amd64)
```

Comment:
are these
```
registry.ng.bluemix.net/armada-master/olm, icr.io/ext/sysdig/agent:latest, registry.ng.bluemix.net/armada-master/metrics-server-amd64
```
private registries ?

Reply:
It's IBM's registry. https://www.ibm.com/cloud/container-registry Any IKS cluster will have these images present.

Reply:
I just setup my IBM cloud account and try to list IBM provided images:
```
$ ibmcloud cr image-list --include-ibm --format "{{.Repository}}" | uniq
icr.io/ibm/alpine
icr.io/ibm/datashield-barbican
icr.io/ibm/datashield-cockroach
icr.io/ibm/datashield-enclaveos-converter
icr.io/ibm/datashield-enclaveos-dkms
icr.io/ibm/datashield-manager-backend
icr.io/ibm/datashield-manager-backend-enclave
icr.io/ibm/datashield-manager-backup
icr.io/ibm/datashield-manager-frontend
icr.io/ibm/datashield-manager-init
icr.io/ibm/datashield-mariadb
icr.io/ibm/datashield-nginx
icr.io/ibm/datashield-node-agent
icr.io/ibm/datashield-node-enclave
icr.io/ibm/datashield-sgx-driver-psw-installer
icr.io/ibm/datashield-vault
icr.io/ibm/hdm/db2graph
icr.io/ibm/hdm/db2rest
icr.io/ibm/ibm-sgx-installer
icr.io/ibm/ibm-sgx-uninstaller
icr.io/ibm/ibm_wa_agent
icr.io/ibm/ibmcloud-alb-metrics-exporter
icr.io/ibm/ibmcloud-backup-restore
icr.io/ibm/ibmcloud-block-storage-attacher
icr.io/ibm/ibmcloud-block-storage-driver
icr.io/ibm/ibmcloud-block-storage-plugin
icr.io/ibm/ibmcloud-cluster-autoscaler
icr.io/ibm/ibmcloud-container-scanner
icr.io/ibm/ibmcloud-event-exporter
icr.io/ibm/ibmcloud-iks-debug
icr.io/ibm/ibmcloud-iks-debug-daemonset
icr.io/ibm/ibmcloud-image-enforcement
icr.io/ibm/ibmcloud-kube-audit-to-logdna
icr.io/ibm/ibmcloud-object-storage-driver
icr.io/ibm/ibmcloud-object-storage-plugin
icr.io/ibm/ibmcloud-secure-perimeter-health
icr.io/ibm/ibmcloud-secure-perimeter-network
icr.io/ibm/liberty
icr.io/ibm/security-advisor-activity-insights
icr.io/ibm/security-advisor-network-insights
icr.io/ibm/sgx-device-plugin
icr.io/ibm/watson-gateway
```
I still think it is quite necessary to whitelisted known images provided by cloud providers `aks`, `gke`, `eks` etc.  Or can you please tell which part I may miss that I don't see the images you listed?

Reply:
If you create an IKS cluster you'll see the images I mentioned present. They can also be pulled. I'm not sure which command would let you list them.

Reply:
let me try what you said. here is the command I listed the IBM images: `ibmcloud cr image-list --include-ibm --format "{{.Repository}}"`

Reply:
@nibalizer, couple comments:
1. Can you make a list `ibm_cloud_containers`
2. Remove the ":latest" tag from `icr.io/ext/sysdig/agent:latest`
3. Update the release note as you've updated the `k8s_container` list, not the `falco_sensitive_mount_containers`

Reply:
Ok Can do.